.gitignore

```diff
@@ -78,3 +78,5 @@ package-lock.json
 
 # plugin history
 .history/
+
+out/
```

Comment:
happy to add in this to gitignore -- can you let me know what this is ignoring?

Reply:
It ended up generating a whole bunch of files which were not relevant to the project so I added it. I don't know why they where there. Instead of adding it when I do a `git add .` and have it show up, I added it to the ignore. 
<img width="1245" alt="Screenshot 2022-10-24 at 10 57 49 AM" src="https://user-images.githubusercontent.com/88281470/197488469-c00e45f4-f406-4013-8d1c-4e6e7d1625fa.png">